### PR TITLE
Checkout

### PR DESCRIPTION
**Pull request by NgocAnhDT**
## Related Tickets:
- [#46246](https://edu-redmine.sun-asterisk.vn/projects/hn_oe49-shoe-store/issues/46246)

## WHAT this PR does?
- Checkout 

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by NgocAnhDT** [comment]
ready

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            $cart = session()->get('cart');
+            if (count($cart) > 0) {
+                return view('users.checkout')->with(compact('cart', 'brands', 'shipping'));
+            } else {
+                Session::flash('noti', 'empty_cart');
+
+                return redirect()->back();
+            }
+        } else {
+            Session::flash('noti', 'empty_cart');
```
Comment (huyenttm-0901) [inline app/Http/Controllers/OrderController.php:30]: sao cái message noti lại như này em?

File: resources/views/users/checkout.blade.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                </div>
+            </div> 
+            <div class="line-space">
+                <button type="submit" class="btn btn-danger float-end mt-3 fs-5 py-1">{{ __('place order') }}</button>
+            </div>
+        </div>
+    </div>
+    </form>
+</div>
+@endsection
```
Comment (huyenttm-0901) [inline resources/views/users/checkout.blade.php:109]: EOF

**Comment by NgocAnhDT** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**